cmd/request.go

```diff
@@ -0,0 +1,145 @@
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/tkhq/tkcli/internal/apikey"
+	"github.com/tkhq/tkcli/internal/clifs"
+	"github.com/tkhq/tkcli/internal/display"
+)
+
+var (
+	requestHost, requestPath, requestBody string
+	requestNoPost, requestShowCurlCommand bool
+)
+
+func init() {
+	makeRequest.Flags().StringVar(&requestHost, "host", "coordinator-beta.turnkey.io", "hostname of the API server")
+	makeRequest.Flags().StringVar(&requestPath, "path", "", "path for the API request")
+	makeRequest.Flags().StringVar(&requestBody, "body", "-", "body of the request, which can be '-' to indicate stdin or be prefixed with '@' to indicate a source filename")
+	makeRequest.Flags().BoolVar(&requestNoPost, "no-post", false, "only provide the signature, do not post the request to the API server")
+	makeRequest.Flags().BoolVar(&requestShowCurlCommand, "show-curl", false, "only provide the signature, do not post the request to the API server")
```

Comment:
This doesn't really tell the difference between the two flags. `show-curl` should be "returns a curl command, ready to be executed". I'm also wondering if we can rename or rework these flags to they compose better (`--show-curl` isn't really compatible with the request being made. It kind of implies `--no-post`)

Reply:
Definitely.  reworked.